### DISCUSSION TITLE
Sum of rows on stat card

### DISCUSSION THREAD

**Discussion by tomy137** [Budibase/budibase#12018] [Help] [answered]
Hello there, 

I'm currently trying Budibase and I can't manage to do something that looks pretty obvious... 

I've inserted a data provider wich contain one number column. And I want, inside this data provider to display the sum of this column named `amount`. 

So I inserted one stat card and write for value : `{{ sum data_provider_name.Rows.amount }}€` but It's always displaying `0€`. I've tried with variations like `{{ sum data_provider_name.Rows amount }}€`  but It is not working. 

What am I doing wrong ? 

Thanks in advance !

**Comment by andz-bb** [comment, marked as answer]
hey @tomy137

The Data Provider rows array contains full row objects. In this case, you need to get an array of just the amounts. You can do this using the [Pluck helper](https://docs.budibase.com/docs/array-helpers#pluck) which returns an array of values for a specified property on each object.

```hbs
{{ pluck data_provider_name.Rows 'amount' }}

// output: 6,8,9,12,123,433,6004
```

You should find that this returns a clean array of just your amounts. Now you can use the sum helper on this array like so:

```hbs
{{ sum (pluck data_provider_name.Rows 'amount') }}

// output: 6595
```

You can also use JavaScript bindings for this functionality which could be more intuitive if you're familiar with JS. Hope this helps!

**Reply by tomy137** [reply]
It's working ! Thanks !
